README.adoc

```diff
@@ -41,4 +41,27 @@ http://openhft.github.io/Chronicle-Map/apidocs/net/openhft/chronicle/map/Chronic
 
 == Chronicle Queue ==
 
-TODO
\ No newline at end of file
+Chronicle Queues can also be configured by setting properties in Spring. These properties correspond with the
+properties the ChronicleQueueBuilder (Indexed & Vanilla) class uses to create and configure a ChronicleQueue.
+Both builders require an instance of a file to be created.
```

Comment:
Can't it be a File instance or a String instance? Spring will call the correct constructor